dhall-yaml/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+1.1.1
+
+* [Add `--generated-comment` flag for `dhall-to-yaml{-ng}`](https://github.com/dhall-lang/dhall-haskell/pull/1840)
+    * You can now optionally add a comment header to the YAML output
+      indicating that the file is generated and should not be hand-edited
```

Comment:
I think this should be duplicated to the `dhall-json` changelog where `dhall-to-yaml` lives.

(Maybe we should discuss consolidating the `dhall-to-yaml{-ng}` situation…)

Reply:
Yeah, I keep forgetting that `dhall-json` contains `dhall-to-yaml` 🙂